src/mp-instance.js

```diff
@@ -59,7 +59,13 @@ export default function mParticleInstance(instanceName) {
     this._instanceName = instanceName;
     this._NativeSdkHelpers = new NativeSdkHelpers(this);
     this._SessionManager = new SessionManager(this);
+
+    // QUESTION: Why can't we just init the store here?
+    // QUESTION: Should we still initialize if persistence is off?
```

Comment:
Do you mean should be initialize the persistence module if persistence is off?  

Short answer is yes for now.

Long answer - I think persistence is too tightly coupled right now, and may be referenced elsewhere...but in a future state this makes sense.  having it around would take up a bit more memory in the browser which while I think we should optimize for, erring on everything working as expected is more important right now. 

For the MVP I was thinking more about finding the points in code that we save/read from persistence, as opposed to turning it off completely, which would be a larger task.

Reply:
I think I'm on the same track as you. I simply wanted to flag this question here so we can use it as a touch point for a conversation of what the relationship between the store and persistence _should_ be so that we can think of an plan for decoupling.